repository/spark/docs/latest/configuration.md

```diff
@@ -75,3 +75,66 @@ purpose KUDO Spark creates a `Role` for them and binds it to a service account p
 If `createRBAC` flag is set to `false`, a `Role` (with a `RoleBinding` linked to Spark Service Account) should be
 created or exist prior to submission of Spark Applications. `Role` configuration and a list of required permissions are
 available in [spark-rbac.yaml](../../operator/templates/spark-rbac.yaml) template file.
+
+### Integration with AWS S3
+This section describes the steps required to provide an access to S3 when using Spark History server 
+or accessing S3 buckets from Spark workloads. 
```

Comment:
Suggestion:

This section describes the steps required for configuring secure access to S3 buckets for Spark workloads and Spark History server.
or accessing S3 buckets from Spark workloads.

Reply:
done